src/main/java/org/folio/config/ApplicationConfig.java

```diff
@@ -590,10 +591,10 @@ PurchaseOrderHelper purchaseOrderHelper(PurchaseOrderLineHelper purchaseOrderLin
         TitlesService titlesService, AcquisitionsUnitsService acquisitionsUnitsService, ProtectionService protectionService,
         PurchaseOrderLineService purchaseOrderLineService, PurchaseOrderStorageService purchaseOrderStorageService,
         ConfigurationEntriesService configurationEntriesService, RestClient restClient,
-        CompositePoLineValidationService compositePoLineValidationService) {
+        CompositePoLineValidationService compositePoLineValidationService, PoLineInvoiceLineHolderBuilder poLineInvoiceLineHolderBuilder) {
     return new PurchaseOrderLineHelper(inventoryManager, encumbranceService, expenseClassValidationService,
```

Comment:
nitpick: Having the constructor having multiple inputs is a code smell that the class is doing to much. No action required.